src/NuGetTrends.Data/Cursor.cs

```diff
@@ -4,7 +4,7 @@ namespace NuGetTrends.Data
 {
     public class Cursor
     {
-        public string Id { get; set; }
+        public string? Id { get; set; }
```

Comment:
This is what sucks when working with EF and null reference types. I really don't like this :(. If you look at this, you might think Id can be null. But then in the entity configuration it cannot. So misleading.

Reply:
Right. Maybe I should make sure it's not null (check the DB and validate before any insert/ `NOT NULL` in the DB

Reply:
Hum, well if you try to insert nulls the DB will throw the exception which is fine in my view, no need to check before or? I like this new feature but I really don't see the value on using this with EF. I also tried on a project and it became so annoying and I had to be checking for nulls all over (when doing queries with `.Include`).. ended up filling with null-forgiving operators and it quickly became a mess. I don't have a solution unfortunately. =/

Reply:
I just dont' want to read NULL (we'll do `null!` in the ctor) so this broken object will travel around the code base where static analysis ensured "non null" everywhere and it'll null ref somewhere.

Reply:
I've opened a new issue for that, I'd like to merge this before. It's too much stuff already #78

Reply:
Fair enough.